uffi-compat/uffi-compat.lisp

```diff
@@ -628,6 +632,12 @@ output to *trace-output*.  Returns the shell's exit code."
           `((with-foreign-strings ,(cdr bindings) ,@body))
           body)))
 
+#+ecl
+(defun foreign-string-length (foreign-string)
+  (declare (inline ffi:foreign-string-length))
+  (ffi:foreign-string-length foreign-string))
```

Comment:
I remember this function used to return a form instead of an integer, does it still do that?